src/js/user-profile/components/FormItem.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import { dateDiffDesc } from '../../common/utils/helpers';
+import { formTitles, formLinks } from '../helpers';
+
+class FormItem extends React.Component {
+  render() {
+    const savedFormData = this.props.savedFormData;
+    const formId = savedFormData.form;
+    const { last_updated: lastSavedTime, expires_at: expirationTime } = savedFormData.metadata;
+    const lastSavedDate = moment.unix(lastSavedTime);
+    const expirationDate = moment.unix(expirationTime);
+    const isExpired = moment(expirationDate).isBefore(moment().startOf('minute'));
```

Comment:
I don't think we need the `startOf('minute')` part. This is the (reversed) check in the intro component:

`moment.unix(f.metadata.expires_at).isAfter()`

Reply:
Good to know!